.gitignore

```diff
@@ -11,3 +11,6 @@ __pycache__
 
 # Linkchecker ignored files
 /.linkchecker
+
+# Mac OS ignored files
+.DS_Store
```

Comment:
This file is now missing a trailing newline.  In one of the follow-on PRs, @JoelMarcey, if you could fix this, that would be good.